guard/src/commands/files.rs

```diff
@@ -79,7 +79,7 @@ where
     }
 }
 
-pub(crate) fn alpabetical(first: &walkdir::DirEntry, second: &walkdir::DirEntry) -> Ordering {
+pub(crate) fn alphabetical(first: &walkdir::DirEntry, second: &walkdir::DirEntry) -> Ordering {
```

Comment:
unrelated to the meat of this PR, should we introduce a CI / IDE implementation for cspell? Would catch things like this. should be fairly low effort. Quickly searching I found this premade action: https://github.com/streetsidesoftware/cspell-action

Reply:
Great suggestion, feel free to create an issue for this, and we can track it :)